In [3]:
n = int(input())
i=1
while i<=n:
  j=1
  while j<=i:
    print(j, end = '')
    j= j+1
  print()
  i= i+1


4
1
12
123
1234


In [6]:
n = int(input())
i=1
while i<=n:
  j=1
  k=i
  while j<=i:
    print(k, end = '')
    j= j+1
    k = k+1
  print()
  i = i+1


5
1
23
345
4567
56789


look for 3rd row i.e 345

* i =3 , 

  j=1 (1 is first column), k=i so k=3 

 so 3

* j+1 so 1+1=2 (2nd column), k+1 so 3+1=4 
  
  so 4

* j+1 so 2+1=3, k+1 so 4+1=5
  
   so 5

   in 3rd column 345

look for 4th row i.e 4567

* i =4 , 

 j=1 (1 is first column), k=i so k=4 

 so 4

* j+1 so 1+1=2 (2nd column), k+1 so 4+1=5
  
  so 5


In [7]:
n = int(input())
i = 1
p = 1
while i<=n:
    j = 1
    while j<=i:
        print(p,end='')
        p = p + 1 # p is 1 so it will keep increment.
        j = j + 1
    print()
    i = i + 1   

5
1
23
456
78910
1112131415


In [ ]:
v